lib/hashgraph.cc

```diff
@@ -311,7 +311,7 @@ consume_fasta_and_tag(
     while (!parser->is_complete( )) {
 
         try {
-            read = parser->get_next_read( );
```

Comment:
I would keep this version and ditch `imprint_next_read()` in the interest of trimming